internal/configs/hcl2shim/merge_body.go

```diff
@@ -3,7 +3,7 @@
 // Copyright (c) 2023 HashiCorp, Inc.
 // SPDX-License-Identifier: MPL-2.0
 
-package configs
+package hcl2shim
```

Comment:
This package desperately needs documentation. There should be a README.md in it as we will likely open it up the same time we move the encryption package into the public. Exported functions also all need documentation please.

Reply:
All of the exported structures/functions have documentation.